File: Python/Lista 3/positivesAndAverage.py
p = t = 0
n = float(input())
if n > 0:
    p +=1
    t += n
for v in range (5):
    n = float(input())
    if n > 0:
        p +=1
        t += n
print(f'''{p} valores positivos
{(t/p):.1f}''')

File: Java/Lista 3/positivesAndAverage.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        float lista[] = new float[6];
        int posi = 0;
        float posimedia = 0;
        for (int i = 0; i < lista.length; i++){
            lista[i] = sc.nextFloat();
            if(lista[i] > 0) {
                posi++;
                posimedia += lista[i];
            }
        }
        float media = (posimedia/posi);
        System.out.println(posi + " valores positivos");
        System.out.printf("%.1f\n", media);
    }
}